metadata-ingestion/src/datahub/ingestion/source/unity/config.py

```diff
@@ -135,7 +135,7 @@ class UnityCatalogSourceConfig(
     )
 
     include_metastore: bool = pydantic.Field(
-        default=True,
+        default=False,
```

Comment:
let's call out that this is a breaking change?

It might also be good to show examples of what the hierarchy would look like when this is true/false

Reply:
also we should say we recommend keeping this set to false, and that folks should use `platform_instance` instead

Reply:
I'll add the section in doc about hierarchy + update description of `include_metastore` config field.

As far as I understand, we'll get rid of this config, eventually. So the section would not be useful.

Reply:
1. added in updating-datahub.md as a breaking change
2. added the examples in updating-datahub doc, I believe, you meant this only  and not to add this in source docs ?
3. we already have pydantic validator for warning if this is not set to false, so we are covered there. changed the error message to represent current state.